odl/util/utility.py

```diff
@@ -417,6 +417,36 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl='general'):
+    """Compute the signal-to-noise ratio.
+    This compute::
+        impl='general'
+            SNR = s_power / n_power
+        impl='dB'
+            SNR = 10 * log10 (
+                s_power / n_power)
+    Parameters
+    ----------
+    signal : projection
+    noise : white noise
+    impl : implementation method
+    """
+    if np.abs(np.asarray(noise)).sum() != 0:
+        ave1 = np.sum(signal)/signal.size
+        ave2 = np.sum(noise)/noise.size
+        s_power = np.sqrt(np.sum((signal - ave1) * (signal - ave1)))
+        n_power = np.sqrt(np.sum((noise - ave2) * (noise - ave2)))
+        if impl == 'general':
+            snr = s_power/n_power
```

Comment:
You should run all tests before you commit, this will fail the PEP8 checker. To run all tests do:

```bash
$ py.test --pep8 --largescale
```

This should be written

```python
snr = s_power / n_power
```